cmd/vfkit/main.go

```diff
@@ -25,54 +25,61 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/Code-Hex/vz/v2"
+	"github.com/Code-Hex/vz/v3"
+	"github.com/crc-org/vfkit/pkg/cmdline"
 	"github.com/crc-org/vfkit/pkg/config"
 	"github.com/crc-org/vfkit/pkg/vf"
 	"github.com/docker/go-units"
 	log "github.com/sirupsen/logrus"
 )
 
-type cmdlineOptions struct {
-	vcpus     uint
-	memoryMiB uint
+func newLegacyBootloader(opts *cmdline.Options) config.Bootloader {
+	if opts.VmlinuzPath == "" && opts.KernelCmdline == "" && opts.InitrdPath == "" {
```

Comment:
Shouldn't this with `||` instead `&&`, if anything of those options are empty string should be nil?

Reply:
Below there is `cmd.MarkFlagsRequiredTogether("kernel", "initrd", "kernel-cmdline")` so if one is set, all must be set. The behaviour was the same before this commit, they were all required (even if that's not 100% correct, kernel should be mandatory and the other optional).
I would keep the && for now.

Reply:
Alright make sense.